images/base/Dockerfile

```diff
@@ -1,17 +1,17 @@
 #
 # This is the base image from which all OpenShift Origin images inherit. Only packages
-# common to all downstream images should be here. Depends on Centos 7.2+.
+# common to all downstream images should be here.
 #
 # The standard name for this image is openshift/origin-base
 #
-FROM centos:centos7
+FROM openshift/origin-source
 
-RUN INSTALL_PKGS="which git tar wget hostname sysvinit-tools util-linux bsdtar epel-release \
-      socat ethtool device-mapper iptables tree findutils nmap-ncat e2fsprogs xfsprogs lsof device-mapper-persistent-data ceph-common" && \
-    yum install -y $INSTALL_PKGS && \
-    rpm -V $INSTALL_PKGS && \
+RUN INSTALL_PKGS="bsdtar ceph-common device-mapper device-mapper-persistent-data e2fsprogs epel-release ethtool findutils git hostname iptables lsof nmap-ncat socat sysvinit-tools tar tree util-linux wget which xfsprogs" && \
+    yum install -y ${INSTALL_PKGS} && \
+    rpm -V ${INSTALL_PKGS} && \
     yum clean all && \
     mkdir -p /var/lib/origin
 
-LABEL io.k8s.display-name="OpenShift Origin Centos 7 Base" \
+LABEL io.openshift.tags="openshift,base" \
```

Comment:
@tdawson did anyone review the tags for catalog impact, or just basic ones?

We can leave in place for now but would like to follow up.